article/app/views/fragments/articleHeaderColumn.scala.html

```diff
@@ -8,9 +8,9 @@
 
 <header class="content__head content__head--splash content__head--article tonal__head tonal__head--@toneClass(article)
 @if(article.content.hasTonalHeaderByline && article.tags.hasLargeContributorImage && article.metadata.designType.nameOrDefault != "analysis") { content__head--byline-pic}">
-    @fragments.mainMedia(article, amp)
+    @fragments.mainMedia(article, false)
 
-    @fragments.meta.metaInline(article, amp)
+    @fragments.meta.metaInline(article, false)
```

Comment:
Maybe `amp = false` is clearer for people reading?

Reply:
This is an intermediary state, I am modifying the signatures in cascade :)

More exactly, in this case I was changing the signature of the view itself, and therefore hardcoded the value `false`, but then next step is to simplify the signature of `@fragments.meta.metaInline`, so that `@fragments.meta.metaInline(article, false)` becomes `@fragments.meta.metaInline(article)` :)